tuftool/src/add_role.rs

```diff
@@ -140,7 +140,7 @@ impl AddRoleArgs {
             .sign(&keys)
             .await
             .context(error::SignRepoSnafu)?;
-        let metadata_destination_out = &self.outdir.join("metadata");
```

Comment:
So as I said where we talked privately:

* Right now, I think it's fair if we just hardcode what we need (let's go the path of least amount of work for now)
* Eventually, I think it would make sense to create a notion of "repository layout" that would be passed in by flag, e.g. `--repo-layout default` vs `--repo-layout sigstore` and all the functions would "query" the selected layout class to see where to put files. Then upstream would default to "default" and we would just do a minor change to default to "sigstore" and keep most of the code upstream.